azure-pipelines.yml

```diff
@@ -38,10 +38,6 @@ jobs:
       # Required when binary dependencies are not bundled
       - attrs==22.2.0    # Fix version. Otherwise tk-ci-tools will install latest
       - Twisted==22.10.0 # We use 22.10.0 to avoid breaking CI for Python 3.7
-      - certifi==2024.7.4
-      - autobahn==22.12.1
-      - pyOpenSSL==24.0.0
-      - service-identity==21.1.0
```

Comment:
Why are we free to clear these ones?

Reply:
They were added so these dependencies can be found by Python environment if bundles were not present.

Introduced when we add Python 3.10 but not binaries for 3.10. Later on, we added 3.10 binaries and we introduced 3.11 but no binaries for it. Now we have both CI running up to 3.11 and bundled binaries.